### PR TITLE
Chefstyle fixes

### PR DESCRIPTION
**Pull request by lamont-granquist**
Fixes:

- Style/HashSyntax
- Layout/IndentHeredoc
- Layout/DocPosition
- Layout/LeadingCommentSpace
- Style/UnneededInterpolation
- Style/TrailingCommandInHashLiteral

note that I tried to do one-fix-per-commit, but some fixes caused other cops to fire which fixes some whitespace issues in the individual commits.

### CODE REVIEW

File: chef-config/spec/unit/workstation_config_loader_spec.rb [hunk 1]
```diff
@@ -389,12 +389,12 @@
 
       context "and has a default profile" do
         let(:content) do
-          content = <<EOH
-[default]
-node_name = 'barney'
-client_key = "barney_rubble.pem"
-chef_server_url = "https://api.chef.io/organizations/bedrock"
-invalid_config_option1234 = "foobar"
+          content = <<~EOH
```
Comment (tas50) [inline chef-config/spec/unit/workstation_config_loader_spec.rb]: This is a very nice rule addition

Comment (lamont-granquist) [reply chef-config/spec/unit/workstation_config_loader_spec.rb]: it made me happy =)

File: lib/chef/application/windows_service.rb [hunk 2]
```diff
@@ -200,8 +200,8 @@ def run_chef_client
           timeout: Chef::Config[:windows_service][:watchdog_timeout],
           logger: Chef::Log
         )
-        Chef::Log.trace "#{result.stdout}"
```
Comment (tas50) [inline lib/chef/application/windows_service.rb]: I guess it's more clear what we're trying to do here, but this seems a bit odd

File: lib/chef/chef_fs/chef_fs_data_store.rb [hunk 3]
```diff
@@ -537,9 +537,9 @@ def list(path)
           if chef_fs.versioned_cookbooks || path[0] == "cookbook_artifacts"
             result = with_entry([ path[0] ]) do |entry|
               # list /cookbooks/name = filter /cookbooks/name-version down to name
-              entry.children.map { |child| split_name_version(child.name) }.
-              select { |name, version| name == path[1] }.
```
Comment (tas50) [inline lib/chef/chef_fs/chef_fs_data_store.rb]: So much better

File: lib/chef/application/windows_service.rb (hunk 2 above)
Comment (lamont-granquist) [reply lib/chef/application/windows_service.rb]: the parens are maybe unnecessary but `result.stdout.to_s` is definitely better than `"#{result.stdout}"`

**Pull request merged**